Top-level Makefile does not know how to build convenience libraries

Unfortunately, this is where my Automake-foo ends, but I think that this should be possible:
```
$ cd chocolate-doom/src
$ make chocolate-heretic
make: *** No rule to make target 'heretic/libheretic.a', needed by 'chocolate-heretic'.  Stop.
```

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -168,6 +168,20 @@ endif
 $(SETUP_BINARIES): @PROGRAM_PREFIX@setup$(EXEEXT)
 	cp @PROGRAM_PREFIX@setup$(EXEEXT) $@
 
+# Make "make" aware of convenience libraries in subdirectories
+
+doom/libdoom.a:
+	$(MAKE) -C doom
+
+heretic/libheretic.a:
+	$(MAKE) -C heretic
+
+hexen/libhexen.a:
+	$(MAKE) -C hexen
+
+strife/libstrife.a:
+	$(MAKE) -C strife
+
 # Source files needed for chocolate-setup:
 
 SETUP_FILES=                               \

